tslint.json

```diff
@@ -70,6 +70,24 @@
             "arrow-parameter",
             "property-declaration",
             "variable-declaration"
-        ]
+        ],
+        "no-magic-numbers": [
+            true
+        ],
+        "no-reference": true,
+        "curly": true,
```

Comment:
Yup, you're right. Yeah the syntax I wrote earlier would be supported with this rule. Good catch :disappointed_relieved: